### ISSUE TITLE
Update the primary navigation menu

### ISSUE THREAD

**Issue by andrewstiefel** [AccelerateNetworks/NumberSearch#206]
**How will this improve revenue or reduce our costs? Please provide a complete explanation!**
Streamlines navigation to help customers find the information they need to make a purchase decision. It also lays the foundation to create a guided checkout flow. This is a step to improve the overall purchase flow.

**Is your feature request related to a problem? Please describe.**
The features page is currently hidden in the about menu, making it hard to quickly find info about the product offering. The current navigation also suggests each page can be browsed in any order when in fact checkout should follow a specific path.

**Describe the solution you'd like**
I would like to update the primary navigation menu to the following:
- Features
- Pricing
- Resources
- Get started [walks customers through checkout flow]

**Additional context**
The long-term goal is to evolve the website content towards the following menu structure:
- Solutions [section with sub-pages targeted to different customer personas; also supports SEO]
- Features [section with sub-pages for primary features; also supports SEO]
- Pricing [clear pricing information and one of the most important questions for customers]
- Resources [support docs, blog, about -- help customers get the most from their investment]

**Comment by uncheckederror** [comment]
The features page is in the about is in the "About" drop down because it's not part of the checkout flow. "Services", "Hardware", "Numbers", and "Cart" are the 4 steps of the checkout flow. The "Support" and "About" dropdowns, as well as the items inside of them, are not part of the checkout flow, which is why they are separated and condensed.

You can initiate the existing guided checkout flow from the homepage by either searching for a phone number or clicking on the "Start Now" call to action. On each page in the checkout flow the user is guided by calls to action in highlight colors and prompted by a sticky footer to continue in the process.

We've discussed doing a solutions or case-studies page in the past but it's never gone anywhere as the sales team hasn't had time to provide the content, yet.

The features page is still WIP as we're waiting on the work in #155 to be completed.

Pricing is tough as each deal is bespoke to the customer and their negotiations with the sales team. We have set starting prices on individual items that you can add to the cart though. Unless the customer knows exactly what they want when they arrive on the website I doubt they'll be able to make it through without chatting with the sales team. Which is why we want to encourage someone who ended up on the website to call the sales team as quickly as possible.

Any thoughts @danry25?
